lib/wickedbase.pm

```diff
@@ -22,9 +22,12 @@ use Carp;
 
 sub wicked_command {
     my ($self, $action, $iface) = @_;
-    my $cmd = q(/usr/sbin/wicked --debug all --log-target syslog ) . $action . ' --timeout infinite ' . $iface;
+    my $cmd = '/usr/sbin/wicked --debug all --log-target syslog ' . $action . ' --timeout infinite ' . $iface;
+    assert_script_run(q(echo -e "\n# ") . $cmd . ' >> /tmp/wicked_serial.log');
```

Comment:
@mtomaschewski WDYT ? does it make value for you to have such isolated log of wicked commands ?

Reply:
@asmorodskyi I did this to avoid searching things in serial_terminal.txt which contains all the commands of all the tests and openqa stuff. This way, we have isolated wicked commands + output in a single file fore each test. This is an example of this file for one test: https://pastebin.com/raw/yaQ433Ly

Reply:
The example file looks very good to me as an overview of the commands + interface status.
I'd just be careful with the `--timeout infinite` use...

Reply:
@mtomaschewski ok, thanks for the feedback. The reason of --timeout infinite is to workaround an openqa issue. If we find a way to remove it we will.

Reply:
@mtomaschewski it is actually "fake" infinite because openQA has it's own timeouts ( default one is 90 seconds ) , so it is just move responsibility for how long command will run from wicked to testing framework.